Assignment2/hxshfx-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,143 @@
+{
+    "@context":{
+
+        "schema":"http://schema.org/",
+        "rdfs":"http://www.w3.org/2000/01/rdf-schema#",
+        "foaf":"http://xmlns.com/foaf/0.1/",
+        "dbp":"http://dbpedia.org/resource/classes#",
+        "npg":"http://ns.nature.com/terms/",
+
+        "person":"foaf:person",
+        "name":"foaf:name",
+        "age":"foaf:age",
+        "city":"dbp:city",
+        "university":"dbp:university",
+        "parent":"dbp:parent",
+        "teacher":"dbp:headteacher",
+        "subject":"npg:subject",
+        "description":"rdfs:comment"
+
```

Comment:
You forgot to define the "parents" attribute.

Reply:
Yes, it is. Your list of elements is a list of resources and they need to be related to another resource. In this assignment, you need to represent that the resource Pablo has Parents that are those resources you defined in this list. Thus, you need to define this relationship with a specific attribute. In this case, you need to define "parents" in the context, it might be something like "parents":"dbp:hasParents".

The same happens with subjects and teachers. Please define both in the context.